### ISSUE TITLE
Training continuation

### ISSUE THREAD

**Issue by Adamei2025** [ultralytics/hub#1203]
### Search before asking

- [x] I have searched the HUB [issues](https://github.com/ultralytics/hub/issues) and [discussions](https://github.com/ultralytics/hub/discussions) and found no similar questions.

### Question

Hello, 

Could you please advise if i have already finished model training at 300 epochs and model is succefully trained. Can i somehow continue the training adding more epochs to further improve it? Or i need to run the new training form the start.

**Comment by pderrenger** [comment]
Hi @Adamei2025 — great question!

Short answer: yes, you can keep training without starting from scratch. You have two good options depending on whether your original run is “finished” or simply “stopped/paused.”

Option A — Resume the same run (best when training was interrupted/paused)
- In Ultralytics HUB: open your Model page and click Resume Training. This restores weights, optimizer, LR scheduler and epoch counter and continues from the last checkpoint. See the HUB Cloud Training notes on resuming from the last checkpoint. See the Ultralytics HUB Cloud Training docs for details.
- CLI/Python (outside HUB), if you have last.pt:
  - CLI: yolo train resume model=path/to/last.pt
  - Python:
    from ultralytics import YOLO
    model = YOLO("path/to/last.pt")
    model.train(resume=True)
  See the Model Training docs (Resuming Interrupted Trainings) for examples.

Option B — Fine‑tune for more epochs after a completed run (recommended in your case)
- In Ultralytics HUB:
  - Go to Models → Train Model → Step 2 “Model” → open the Custom tab and select your previously trained model as the starting point (best.pt). Set the new number of epochs and start training. This creates a new run that continues learning from your trained weights. See the Ultralytics HUB Models guide (Custom tab and continuing from a previous model).
- CLI/Python (outside HUB):
  - CLI: yolo detect train model=path/to/best.pt data=your_data.yaml epochs=50
  - Python:
    from ultralytics import YOLO
    model = YOLO("path/to/best.pt")
    model.train(data="your_data.yaml", epochs=50)
  This is standard fine‑tuning and does not reset the weights; it simply continues learning with a fresh optimizer schedule.

Practical tips
- If you’re extending training, consider lowering lr0 slightly and using patience for early stopping to avoid overfitting. See the training tips guide for suggestions.
- Checkpoints are saved every epoch by default, so you can always resume if a session stops mid‑run. See the Model Training docs.

Links
- See the Ultralytics HUB Models guide for continuing from a previous model.
- See the Ultralytics HUB Cloud Training docs for resuming from checkpoints and billing behavior during paused runs.
- See the Model Training docs (Resuming Interrupted Trainings) for CLI/Python resume examples.
- See the training tips guide for practical advice on epochs, learning rate, and early stopping.

If you share your HUB model URL (read‑only is fine), I’m happy to point you to the exact Resume/Train buttons in your project UI.

**Comment by Adamei2025** [comment]
Hi @pderrenger thank you!

I have initated the pricess but seems it is stuck. Can you please help to restore: https://hub.ultralytics.com/models/sC1om9Oh62Onmz5JHHPB

**Comment by yogendrasinghx** [comment]
Hi @Adamei2025,

Thanks for sharing the details. Our team is aware of the issue and is already working on a fix. I’ll update you here once it’s resolved.

Apologies for the inconvenience, and thank you for your patience!
